app/views/addresses/edit.haml

```diff
@@ -0,0 +1,19 @@
+= content_for :title, 'Adresse bearbeiten'
+
+.page-header
+  %h1= content_for :title
+
+= bootstrap_form_for @address,
+  layout: :horizontal,
+  label_col: 'col-md-2',
+  control_col: 'col-md-4' do |f|
+  = f.hidden_field :addressable_id
```

Comment:
I'm wondering why we just pass `:addressable_id` and not `:addressable_type` too. but at the moment we are only updating addresses with this form, right? how is an address created then?